ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -168,23 +174,27 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
                     .lineLimit(1)
                     .foregroundColor(Color(dynamicColor: foregroundColor))
                     .font(.fluent(tokens.boldTextFont))
+                    .hoverEffect()
                 } else {
                     SwiftUI.Button(action: {
                         isPresented = false
                         buttonAction()
                     }, label: {
                         if let trailingImage = state.trailingImage {
                             Image(uiImage: trailingImage)
+                                .accessibilityLabel(state.trailingImageAccessibilityLabel ?? "")
```

Comment:
Shouldn't we have a default accessibility label for the trailing image instead of an empty string?

Reply:
By default, the notification will use an X icon and have "Dismiss" as an accessibility label. This is only for if they've set a custom trailing image.

Reply:
Ah, but, that's not quite what you were suggesting. This matches just the normal default behavior of an image, I think, and lets us add the label if there is one. If `.modifyIf` worked with `if let`, I think that would've been a better choice, but it doesn't. So with this approach we either have the regular default behavior, or the desired label.